### ISSUE TITLE
`disk` event is emitted on startup

### ISSUE THREAD

**Issue by Ale32bit** [SquidDev-CC/CC-Tweaked#1722]
### Minecraft Version

1.20.x

### Version

1.109.3

### Details

The event `disk` is emitted on startup if an attached disk drive contains a disk, despite not interacting with it at all.

To reproduce attach a disk drive to a computer and insert a disk, then restart the computer.

The following startup.lua file waits for the `disk` event to be emitted, which normally happens when you insert something in the drive, but unfortunately it immediately happens at every start.

```lua
print(os.pullEvent("disk"))
print("os.clock() =", os.clock())
print("disk event was fired")
```

**Comment by SquidDev** [comment]
Can you update to 1.109.5 and try again? This should have been fixed in f695f22d8a6ae5987ad59ac69489071a4ab6e47b.

**Comment by Ale32bit** [comment]
Yeah it's fixed, now to convince the server admins to update. Thanks
